### PR TITLE
fix: config pip source speed up in china

### PR DESCRIPTION
**Pull request by kenwoodjw**

### CODE REVIEW

**Comment by gaocegege** [comment]
Thanks for the PR, but users in US may suffer. :thinking:

**Comment by kenwoodjw** [comment]
> Thanks for the PR, but users in US may suffer. 🤔

maybe write tips in docs?

**Comment by gaocegege** [comment]
Actually, you can set the pip mirror in `~/.config/envd/config.envd` (in macOS it should be `~/Library/Application Support/envd/config.envd`) like this:

```
config.apt_source(source="""
# 默认注释了源码镜像以提高 apt update 速度，如有需要可自行取消注释
deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy main restricted universe multiverse
# deb-src https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy main restricted universe multiverse
deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy-updates main restricted universe multiverse
# deb-src https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy-updates main restricted universe multiverse
deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy-backports main restricted universe multiverse
# deb-src https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy-backports main restricted universe multiverse
deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy-security main restricted universe multiverse
# deb-src https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy-security main restricted universe multiverse

# 预发布软件源，不建议启用
# deb https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy-proposed main restricted universe multiverse
# deb-src https://mirrors.tuna.tsinghua.edu.cn/ubuntu/ jammy-proposed main restricted universe multiverse
""")
config.pip_index(url = "https://pypi.tuna.tsinghua.edu.cn/simple")
git_config(name="Name", email="email", editor="vim")
```

The instructions in `~/.config/envd/config.envd` will be run in every `envd build`

File: examples/mnist/build.envd [hunk 1]
```diff
@@ -3,7 +3,7 @@ def build():
     install.vscode_extensions([
         "ms-python.python",
     ])
-
+    config.pip_index(url = "https://pypi.tuna.tsinghua.edu.cn/simple")
```
Comment (gaocegege) [inline examples/mnist/build.envd:6]:
```suggestion
    # config.pip_index(url = "https://pypi.tuna.tsinghua.edu.cn/simple")
```

How about this

Comment (gaocegege) [reply examples/mnist/build.envd:6]: Sure, please reopen it! Thanks for the contribution

**Comment by gaocegege** [comment]
@all-contributors please add @kenwoodjw  for code

**Pull request merged**